english/dog-eat/dog-eat.md

```diff
@@ -0,0 +1,18 @@
+You notice that the walls are beginning to melt, but fortunately
```

Comment:
I didn't (and still don't) understand how to get to that file. I ended up doing this entire story using the GitHub GUI...cloning entire create-your-own-adventure repo, including all of the language directories, then all the directories with in them seems...like a lot? Like won't that be just the craziest file structure ever/isn't that a lot more information? If I just need the English directory, how do I (can I?) pull down just that directory? To be honest, I was really really lost in this exercise.